integration/tests/cook/test_cli.py

```diff
@@ -79,9 +79,8 @@ def test_wait_for_multiple_jobs(self):
         cp = cli.wait(uuids, self.cook_url)
         self.assertEqual(0, cp.returncode, cp.stderr)
         cp, jobs = cli.show_json(uuids, self.cook_url)
-        self.assertEqual(uuids[0], jobs[0]['uuid'])
-        self.assertEqual(uuids[1], jobs[1]['uuid'])
-        self.assertEqual(uuids[2], jobs[2]['uuid'])
+        returned_uuids = [j['uuid'] for j in jobs]
+        self.assertEqual(sorted(uuids), sorted(returned_uuids))
```

Comment:
I think this sorting can go away if you maintain the order of the uuids internally.

Reply:
Yep, done